libs/video.liq

```diff
@@ -39,15 +39,15 @@ end
 # @param input ffmpeg options specifying the input
 def input.external.ffmpeg(~id="", ~show_command=false, input)
```

Comment:
Please wrap up the other parameters from `output.external.avi`, in particular `fallible`..